src/cards/potion_shatter.ts

```diff
@@ -0,0 +1,59 @@
+import { CardCategory } from '../types/commonTypes';
+import { playDefaultSpellSFX } from './cardUtils';
+import { Spell, refundLastSpell } from './index';
+import { CardRarity, probabilityMap } from '../types/commonTypes';
+import { baseExplosionRadius, explode } from '../effects/explode';
+import { removePickup } from '../entity/Pickup';
+import { drawUICirclePrediction } from '../graphics/PlanningView';
+import * as colors from '../graphics/ui/colors';
+import { COLLISION_MESH_RADIUS } from '../config';
+
+export const potionShatterId = 'Potion Shatter';
+const baseEffectRadius = 50; //baseExplosionRadius / x?
+const spell: Spell = {
+  card: {
+    id: potionShatterId,
+    category: CardCategory.Blessings,
+    supportQuantity: true,
+    manaCost: 15,
+    healthCost: 0,
+    expenseScaling: 1,
+    probability: probabilityMap[CardRarity.SPECIAL],
+    thumbnail: '',
+    sfx: '',
+    description: ['spell_potion_shatter', baseEffectRadius.toString()],
+    effect: async (state, card, quantity, underworld, prediction) => {
+      // .filter: only target potions
+      const targets = state.targetedPickups.filter(p => p.name.includes("Potion"));
```

Comment:
I believe this will fail to work with non-english localizations.  I'd use a list of Ids instead

Reply:
Right! Forgot to update that. Thanks!